src/mame/drivers/galaxian.cpp

```diff
@@ -8893,6 +8893,28 @@ ROM_START( pacmanbla ) /* content is the same as the above bootleg, but arranged
 	ROM_LOAD( "sn74s288n.6l", 0x0000, 0x0020, CRC(24652bc4) SHA1(d89575f3749c75dc963317fe451ffeffd9856e4d) ) /* same as pisces */
 ROM_END
 
+ROM_START( pacmanbv ) /* Video Dens */
+	ROM_REGION( 0x10000, "maincpu", 0 )
```

Comment:
Mainly because I'm assuming that, like on the other sets, there is an unpopulated area. I'm not really sure, so I have not added a comment, but sized the area as if it's like that.